pkg/collector/corechecks/oracle-dbm/config/config.go

```diff
@@ -235,6 +236,14 @@ func NewCheckConfig(rawInstance integration.Data, rawInitConfig integration.Data
 		}
 	}
 
+	if instance.Username == "" {
+		if instance.User != "" {
```

Comment:
Assume `user` is a deprecated config. let's add a warning message?

Reply:
I changed it.